app/src/main/java/com/breadwallet/BreadApp.java

```diff
@@ -183,25 +149,35 @@ public void run() {
                 } catch (GooglePlayServicesNotAvailableException e) {
                     e.printStackTrace();
                 }
-
                 finished(null);
             }
         });
-
         thr.start();
     }
 
     private void finished(final AdvertisingIdClient.Info adInfo) {
         if(adInfo!=null) {
-            adInfoString = adInfo.getId();
-            //long timestamp = new Date().getTime()
-            Timber.d("timber: Finished Ad ID : %s -", adInfoString);
-            ////adInfoString =
+
+            String adInfoString = adInfo.getId();
+            String generalAndroidInterest = "general-android";
+            String debugGeneralAndroidInterest = "debug-general-android";
+            String pusherInstanceID = Utils.fetchPartnerKey(this, PartnerNames.PUSHERSTAGING);
+
+            // setup Push Notifications
+            //This worked had to add the iid dep https://github.com/mixpanel/mixpanel-android/issues/744
+            PushNotifications.start(getApplicationContext(), pusherInstanceID);
+            PushNotifications.addDeviceInterest(generalAndroidInterest);
```

Comment:
What is addDeviceInterest?

Reply:
these are a way for pusher to send to different groups. For instance we might want to push to Spanish speaking people. The push would be in Spanish and we might add the interest `español-android-interest` & `español-ios-interest` and just send pushes to them

Reply:
thanks for the explanation